src/Driver/Driver.jl

```diff
@@ -290,6 +294,11 @@ function parse_commandline(
         metavar = "<number>"
         arg_type = Int
         default = get_setting(:fixed_number_of_steps, defaults, global_defaults)
+        "--degree"
+        help = "tuple of horizontal and vertical polynomial degrees for spatial discretization order (no space before/after comma)"
+        metavar = "horizontal,vertical"
```

Comment:
```suggestion
        metavar = "<horizontal>,<vertical>"
```
For consistency with our other arguments' metavars.

Reply:
Thanks for catching it. Updated it